accounts/admin.py

```diff
@@ -0,0 +1,16 @@
+from django.contrib import admin
+
+from .models import Profile
+
+
+class ProfileAdmin(admin.ModelAdmin):
+    # Defining fields for Admin panel
```

Comment:
Do we also not want to add a docstring instead of a single line comment on `class` ?